hexstody-public/src/api/auth.rs

```
@@ -41,7 +40,12 @@ pub async fn signup_email(
         if let Some(_) = mstate.users.get(&data.user) {
             return Err(error::Error::SignupExistedUser.into());
         } else {
-            let body = reqwest::get(&("http://localhost:8000/createuser/".to_owned()+&data.user)).await.unwrap().text().await;
+            reqwest::get("http://localhost:8000/createuser/".to_owned() + &data.user)
+                .await
+                .unwrap()
```

I think it's better to return an error than to panic.